Classes/Controllers/PBGitHistoryController.m

```diff
@@ -180,11 +180,10 @@ - (void) _repositoryUpdatedNotification:(NSNotification *)notification {
 - (void)reselectCommitAfterUpdate {
 	[self updateStatus];
 
-	if ([self.repository.currentBranch isSimpleRef]) {
-		[self selectCommit:[self.repository OIDForRef:self.repository.currentBranch.ref] withRescroll:NO];
-	} else {
-		[self selectCommit:self.firstCommit.OID withRescroll:NO];
-	}
+	if ([self.repository.currentBranch isSimpleRef])
+		[self selectCommit:[self.repository OIDForRef:self.repository.currentBranch.ref]];
+	else
+		[self selectCommit:self.firstCommit.OID];
```

Comment:
is `if () ; else ;` correct? I'd prefer to always see braces, but I'm just a random dude on the internet.

Reply:
It is. In this case, the braces are clearly optional, and as it keeps "visual noise" down I tend to prefer that. But I'm just another random dude who happens to have commit access 😜. Also, it doesn't help that I'm writing code in at least 4 different, project-dependent, syntax styles, so…

To be clear, I suspect the refresh bug of #191 is caused by this condition being false at an unexpected time — that statement *force-selects* the first commit of the repo, usually `HEAD`, which matches the bug. Why, and what causes it to fail, is still a mystery, though I have a hunch `currentBranch` is one of the things that gets reset to `nil` on a refresh ^^.